src/tests/cxxfrontend.c

```diff
@@ -227,6 +227,14 @@ void test_semantic()
     semantic2(m, NULL);
     semantic3(m, NULL);
 
+    Loc loc;
+    Dsymbol *s = m->search(loc, Identifier::idPool("Error"));
+    assert(s);
+    AggregateDeclaration *ad = s->isAggregateDeclaration();
+    assert(ad->ctor);
+    CtorDeclaration *ctor = ad->ctor->isCtorDeclaration();
+    assert(0 == strcmp(ctor->type->toChars(), "Error(string)"));
```

Comment:
Maybe also:
assert(ctor->isMember() && !ctor->is Nested());

To test a couple more virtuals at random intervals in the vtable.